lbry/lbry/extras/daemon/Components.py

```diff
@@ -112,6 +108,7 @@ def __init__(self, component_manager):
         self.headers_dir = os.path.join(self.conf.wallet_dir, 'lbc_mainnet')
         self.headers_file = os.path.join(self.headers_dir, 'headers')
         self.old_file = os.path.join(self.conf.wallet_dir, 'blockchain_headers')
+        self.headers = Headers(self.headers_file)
         self._downloading_headers = None
```

Comment:
I think this can be `self.is_downloading_headers`, is there a reason it needs to be private? also, adding `is_` makes it clear it's always a boolean. default value should be `False`.